### DISCUSSION TITLE
Recurso: Curso de Cyberseguridad

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [AndreaGriffiths11/LatinX-Experts-2022#3] [Announcements]
Sorteo

**Comment by OscarJ290** [comment]
OscarJ290

**Comment by ffarellano** [comment]
@ffarellano

**Comment by erykzon** [comment]
@erykzon

**Comment by charlyguz** [comment]
Yo quiero soy @charlyguz https://github.com/charlyguz

**Comment by BrunoGGM** [comment]
Hey, yo quiero swag @brunoggm

**Comment by malibb** [comment]
@malibb Malinali Becerril

**Comment by RodrigoRiestra** [comment]
rodrigo_riestra

**Comment by DanRoal** [comment]
Daniel Rodríguez Almanza

**Comment by VictorHugoChavez** [comment]
Victor Hugo Chavez @VictorHugoChavez

**Comment by CinthyaAdriana** [comment]
Cinthya Adriana Méndez Barrera 
@CinthyaAdriana

**Comment by MariaEmiliaRG** [comment]
Holiiii ✨✨
MariaEmiliaRG

**Comment by amargomxn** [comment]
@amargomxn

**Reply by amargomxn** [reply]
Muchas gracias por la oportunidad

**Comment by AlamillaAx** [comment]
@AlamillaAx

**Comment by JosAnGoCa** [comment]
@JosAnGoCa

**Comment by PaolaDaft** [comment]
@PaolaDaft

**Comment by rosaainz** [comment]
@rosaainz

**Comment by cristhianmt** [comment]
Cristhianmt

**Comment by TristanDom** [comment]
Hola, soy Tristán, entusiasta de la seguridad y la inteligencia artificial:)

**Comment by JaquelineRazo** [comment]
Hola!! Quiero participar :D
[ ](https://github.com/JaquelineRazo)

**Reply by JaquelineRazo** [reply]
https://github.com/JaquelineRazo

**Comment by Mike-std-cpu** [comment]
¡Hola a todos!
Quisiera participar 🚀
[@Mike-std-cpu](https://github.com/Mike-std-cpu)

**Comment by AndreaGriffiths11** [comment]
Discussion está cerrada! Todos los que comentaron entran al curso.
